### ISSUE TITLE
$.noConflict(true); Uncaught TypeError: Cannot read property 'easing' of undefined

### ISSUE THREAD

**Issue by markusfalk** [darsain/sly#171]
When using Sly with jquery's noConflict I get an error.

```javascript
// causes: Uncaught TypeError: Cannot read property 'easing' of undefined
$.noConflict(true);
```
I use requirejs (http://requirejs.org/docs/jquery.html#noconflictmap) but I don't think this is the root of the problem.

This is the line that has the undefined jQuery.
```javascript
pos.cur = animation.from + animation.delta * jQuery.easing[o.easing](animation.time/o.speed, animation.time, 0, 1, o.speed);
```

Possible Solution? - Use the private $ instead of 'jQuery':

```javascript
pos.cur = animation.from + animation.delta * $.easing[o.easing](animation.time/o.speed, animation.time, 0, 1, o.speed);
```

**Comment by darsain** [comment]
Good catch. I'l look at it when I'll have some time.

**Comment by darsain** [comment]
Fixed in cbfc3e09fc8ec899aa3ed60db303663679f3987e.
